js/libs/form.js

```diff
@@ -134,6 +134,8 @@ Fliplet.Widget.instance('form-builder', function(data) {
           }
         });
 
+        this.$forceUpdate();
```

Comment:
I'm a bit scared by this. It means the form will be refreshed every single time one of its input changes?